changelog.txt

```diff
@@ -0,0 +1,2 @@
+feature - Experimental: add ability to locally emulate HTTPS functions with "firebase serve --only functions". (Run "firebase serve --only functions,hosting" to emulate both functions and hosting.)
+feature - Add ability to target specific functions and export groups when deploying functions with "firebase deploy --only functions:function1,functions:groupA.function2".

```

Comment:
Let's not talk up groups here since they're not really well-documented anyway.

    feature - Adds ability to target specific functions when deploying via "firebase deploy --only functions:functionname"